tests/robustness/model/describe.go

```diff
@@ -223,6 +223,8 @@ func describeRangeRequest(opts RangeOptions, revision int64) string {
 		return fmt.Sprintf("get(%q%s)", opts.Start, kwargsString)
 	case opts.End == clientv3.GetPrefixRangeEnd(opts.Start):
 		return fmt.Sprintf("list(%q%s)", opts.Start, kwargsString)
+	case strings.HasSuffix(opts.Start, "\x00"):
+		return fmt.Sprintf("continueList(%q%s)", strings.TrimRight(opts.End, "0\x00"), kwargsString)
```

Comment:
Would prefer to see the opts.Start (without "\x00") over the opts.End here.

Reply:
Example from the current version: `list[continued]("/registry/pods/default/dCRep", rev=430, limit=10) -> ["459","454","448","455"], count: 4, rev: 948`